### PR TITLE
update docs to reflect skills versioning

### PR DESCRIPTION
**Pull request by andhesmi**

### CODE REVIEW

**Review by gliu6 — COMMENTED** [review]
better will the desired enrichment change in this pr?

File: ResumeParseAndNormalize.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following parameters may be supplied in the query string (for HTTP GET) or f
...
...** (Required) -- A comma-separated list of the desired normalization calls to perform on the results of the resume parsing operation. The list of possible values is as follows (case-insensitive): **company_norm, geocoding, job_level, job_title_carotene, job_title_onet, school_norm, skillsv4, related_skillsv4, skillsv5, related_skillsv5**. For example, a request with a desired_enrichments value equal to **job_level,skillsv5,job_title_onet,company_norm** would receive job level classifications, skills extractions for version 5.0, ONet job title classifications, and company normalizations. **related_skillsv5** and **skillsv5** differ in that requesting **skillsv5** returns only extracted skills, while requesting **related_skillsv5** returns related skills alongside extracted skills in the document skills node. The **related_skillsv5** enrichment includes the **skillsv5** enrichment; there is no need to also request **skillsv5** when requesting **related_skillsv5**. The same applies to ***skillsv4*** and ***related_skillsv4***. If no additional enrichments are needed, the value **"none"** may be supplied to skip all post-parsing classifications and simply return the parsed resume data.
```
Comment (gliu6) [inline ResumeParseAndNormalize.md:57]: I know when supplying the values to the param, there is no space between two enrichments.
but for this documentation, to make it look better, I would still like to put space in the sentence
"with a desired_enrichments value equal to ** job_level,skillsv5,job_title_onet,company_norm**"

**Review by thomasmahoney — APPROVED** [review]

**Comment by thefroatgt** [comment]
Hey,
You should be paying attention.  Or if not, don’t broadcast that fact to the team, it sets a bad example.

Adam Edwards
Director, Sourcing and Integration Software

**Comment by gliu6** [comment]
👍

**Review by glpayson — APPROVED** [review]

**Pull request merged**